.github/pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+# <Name of PR>
+
+## Description
+<!-- Add Brief description of what this PR does and why it is needed. -->
+
+
+## Testing Instructions
+<!-- Add bullet points for how to test this PR -->
+<!-- * What is the desired goal one can test again -->
+<!-- * How to test against the desired goal -->
+<!-- * Optionally add dry-run instructions -->
+
+
+## How to roll out
+<!-- Add information for how to roll out this PR -->
+<!-- * How to roll out (cli or CI/CD tools) -->
+<!-- * Where to roll out (affected environments) -->
```

Comment:
This line is irrelevant in the context of this being a module and also an open-source one.

Reply:
agreement for PRs are project dependent and nothing to do opensource. I think we can do it as we want